p4-16/spec/P4-16-spec.mdk

```diff
@@ -3569,6 +3569,10 @@ Variable-length bit-strings support a limited set of operations:
   Two `varbit` fields can be compared only if they have the same type.
   Two varbits are equal if they have the same dynamic width and all
   the bits up to the dynamic width are the same.
+- Initialization of a `packet_in` object from a varbit value, by using the
+  `initialize` method of a packet_in extern instance.
+  This allows using the traditional methods of `packet_in` to be used
+  for parsing the data in a varbit field.
```

Comment:
I am a little worried about a variety of cases, where 
1. `varbit` (or any other source field is not byte-aligned)
2. When `varbit` length (in bits) is not divisible by 8
3. What happens when not the whole field is parsed (esp. how will deparsing be defined)

Reply:
These cases are not different from parsing a packet.
Deparsing is not defined in this proposal, but it could be defined by attaching a varbit to a packet_out object. 
One would have to specify what happens on varbit overflow.